### PR TITLE
Change the hash generation algo

### PR DESCRIPTION
**Pull request by vivek378521**

### CODE REVIEW

File: utils/helper.go [hunk 1]
```diff
@@ -1,14 +1,30 @@
...
-	"time"
 )
 
 func VideoIDGen(fileExtension string) string {
 	// Seed the random number generator
-	rand.Seed(time.Now().UnixNano())
+	hash := crc64.New(crc64.MakeTable(crc64.ECMA))
+
+	// Write the string to the hash
+	_, err := hash.Write([]byte("zestream"))
```
Comment (abskrj) [inline utils/helper.go]: correct me if I am wrong, but value of hash will always be same?

Comment (vivek378521) [reply utils/helper.go]: Yes, you are right. I took the wrong approach. Should have just generated random numbers. I have made those changes and updated the PR. Apologies for any inconvenience.

File: controllers/helper_test.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package controllers
```
Comment (uzaxirr) [inline controllers/helper_test.go]: this file `controllers/helper_test.go` should be in the `utils` directory

**Review by uzaxirr — CHANGES_REQUESTED** [review]

File: controllers/helper_test.go (hunk 2 above)
Comment (vivek378521) [reply controllers/helper_test.go]: done

**Review by uzaxirr — APPROVED** [review]
LGTM!

**Pull request merged**